script/e2e-test.sh

```diff
@@ -97,4 +97,27 @@ echo "All deployments ready. PODs:"
 kubectl get pods -n kubeapps -o wide
 
 # Run helm tests
-helm test ${HELM_CLIENT_TLS_FLAGS} --cleanup kubeapps-ci
+set +e
+
+helm test ${HELM_CLIENT_TLS_FLAGS} kubeapps-ci
+code=$?
+
+set -e
+
+if [[ "$code" != 0 ]]; then
+  echo "PODS status on failure"
+  kubectl get pods -n kubeapps
+  for pod in $(kubectl get po -l release=kubeapps-ci -oname -n kubeapps); do
```

Comment:
Same here about exposing sensitive data

Reply:
in this case there is nothing sensitive for the temporary cluster